parser/object_parse/die_parse/item_kind.rs

```diff
@@ -0,0 +1,133 @@
+        // Subroutine call-site tags.
+        gimli::DW_TAG_inlined_subroutine => None,
+        gimli::DW_TAG_call_site => None,
+        gimli::DW_TAG_call_site_parameter => None,
+        gimli::DW_TAG_entry_point => None,
+        // Label entries. (Section 3.6)
+        gimli::DW_TAG_label => unimplemented!(),
+        // With statements. (Section 3.7)
+        gimli::DW_TAG_with_stmt => unimplemented!(),
+        // Data Object and Object List Entries: (Chapter 4)
+        // --------------------------------------------------------------------
+        // Data object entries. (Section 4.1)
+        gimli::DW_TAG_variable | gimli::DW_TAG_formal_parameter => None,
+        gimli::DW_TAG_constant => unimplemented!(),
+        // Common block entries. (Section 4.2)
+        gimli::DW_TAG_common_block => unimplemented!(),
+        // Namelist entries. (Section 4.3)
+        gimli::DW_TAG_namelist => unimplemented!(),
+        gimli::DW_TAG_namelist_item => unimplemented!(),
+        // Type Entries: (Chapter 5)
+        // --------------------------------------------------------------------
+        // Base type entries. (Section 5.1)
+        gimli::DW_TAG_base_type => None,
+        // Unspecified type entries. (Section 5.2)
+        gimli::DW_TAG_unspecified_type => None,
+        // Type modifier entries. (Section 5.3)
+        gimli::DW_TAG_atomic_type => None,
+        gimli::DW_TAG_const_type => None,
+        gimli::DW_TAG_immutable_type => None,
+        gimli::DW_TAG_packed_type => None,
+        gimli::DW_TAG_pointer_type => None,
+        gimli::DW_TAG_reference_type => None,
+        gimli::DW_TAG_restrict_type => None,
+        gimli::DW_TAG_rvalue_reference_type => None,
+        gimli::DW_TAG_shared_type => None,
+        gimli::DW_TAG_volatile_type => None,
+        // Typedef entries. (Section 5.4)
+        gimli::DW_TAG_typedef => None,
+        // Array type entries. (Section 5.5)
+        gimli::DW_TAG_array_type => None,
+        // Coarray type entries. (Section 5.6)
+        gimli::DW_TAG_coarray_type => None,
+        // Structure, union, and class type entries. (Section 5.7.1)
+        gimli::DW_TAG_class_type => None,
+        gimli::DW_TAG_structure_type => None,
+        gimli::DW_TAG_union_type => None,
+        // Interface type entries. (Section 5.7.2)
+        gimli::DW_TAG_interface_type => None,
+        // Derived or extended structures, classes, and interfaces. (Section 5.7.3)
+        gimli::DW_TAG_inheritance => unimplemented!(),
+        // Access declarations. (Section 5.7.4)
+        gimli::DW_TAG_access_declaration => None,
+        // Friend entries. (Section 5.7.5)
+        gimli::DW_TAG_friend => unimplemented!(),
+        // Data member entries. (Section 5.7.6)
+        gimli::DW_TAG_member => None,
+        // Variant entries. (Section 5.7.10)
+        gimli::DW_TAG_variant => None,
+        gimli::DW_TAG_variant_part => None,
```

Comment:
I don't think we should do an exclusive match here, because we explicitly *don't* care about most of these things. It would be easier to do:

```rust
match die.tag() {
    // Match things we care about, and do what we will do with them.
    gimli::DW_TAG_something_we_care_about => do_stuff(die),

    // And then just skip everything else.
    _ => return None,
}
```